pooch/tests/test_downloaders.py

```diff
@@ -48,6 +48,21 @@
 ZENODOURL = pooch_test_zenodo_url()
 
 
+@pytest.mark.skipif(tqdm is None, reason="requires tqdm")
```

Comment:
Needed to skip this test if tqdm is not installed since it's an optional dependency (was causing some CI runs to fail).